coinstacks/bitcoin/indexer/Makefile

```
@@ -0,0 +1,11 @@
+IMAGE = blockbook
+PACKAGER = $(shell id -u):$(shell id -g)
+
+gen-config:
+	- mkdir -p config
+	- docker run --rm -t -e PACKAGER=$(PACKAGER) -v "$(CURDIR):/out" $(IMAGE) ./generate-config.sh ethereum
+
+run:
+	- mkdir -p db
+	- mkdir -p logs
+	- docker run --rm -t -v "$(CURDIR)/config:/config" -v "$(CURDIR)/db:/db" -v "$(CURDIR)/logs:/logs" -p 8000:8000 -p 8001:8001 $(IMAGE) /bin/blockbook -blockchaincfg=/config/local.json -datadir=/db -sync -internal=:8000 -public=:8001 -log_dir=/logs
```

Need to fix this for ethereum as well
```suggestion
	- docker run --rm -t -v "$(CURDIR)/config.json:/config.json" -v "$(CURDIR)/db:/db" -v "$(CURDIR)/logs:/logs" -p 8000:8000 -p 8001:8001 $(IMAGE) /bin/blockbook -blockchaincfg=/config.json -datadir=/db -sync -internal=:8000 -public=:8001 -log_dir=/logs
```